charts/kubechecks/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: kubechecks
 description: A Helm chart for kubechecks
-version: 0.4.6
```

Comment:
No need to bump chart version, we're not making any modifications to the chart

Reply:
I remember now why I had to, it is because the CI throws an error otherwise: https://github.com/zapier/kubechecks/actions/runs/12284880839/job/34281659217?pr=320